facia/conf/application.conf

```diff
@@ -6,6 +6,9 @@
 "assets.cache./public/security.txt"="public, max-age=900"
 "assets.cache./public/security.txt.asc"="public, max-age=900"
 
+# Caching for applepay merchantId file
+"assets.cache./public/apple-developer-merchantid-domain-association-code.txt"="public, max-age=900"
```

Comment:
I’m not entirely sure how the `max-age` property will be interpreted by our CDN. Is there any risk if it’s cached for longer than 15 minutes?

Reply:
Max age is respected from app => router => fastly.
I think on average `frontend` controls most the cache times.